flocx_market/tests/commit/test_commit_message.py

```diff
@@ -0,0 +1,94 @@
+# pylint: disable=missing-docstring,redefined-outer-name,no-self-use,invalid-name
+
+import os
+import re
+import subprocess
+
+import git
+import pytest
+
+RE_PARSE_NAME = re.compile(r'(?P<name>.*) <(?P<email>[^>]+)> '
+                           r'(?P<timestamp>\d+) (?P<tz>[-\d]+)')
+
+
+@pytest.fixture
+def commit_id():
+    '''A dummy fixture that will be parametrized by the pytest_generate_tests
+    function.'''
+
+
+@pytest.fixture
+def repo():
+    '''Return a git.Repo object for the current directory'''
+    return git.Repo('.')
+
+
+@pytest.fixture
+def commit(repo, commit_id):
+    '''Convert the parametrized commit_id into a Commit object'''
+    return repo.commit(commit_id)
+
+
+@pytest.fixture
+def max_subject_length():
+    return int(os.environ.get('COMMIT_MAX_SUBJECT_LENGTH', 60))
```

Comment:
I don't hate this, but am curious - is there a use case where you see a user needing to change the max length?

Reply:
I figured this number would be open to bikeshedding, and having it set via an environment label means we can just update the travis job rather than pushing changes.